zoo-hope/src/components/footer/index.tsx

```diff
@@ -15,25 +26,25 @@ export const Footer = () => {
           <p className="footer__logo-text">zoonadia</p>
         </div>
         <div className="footer__navigation">
-          {elements.map((e:Object) =>
+          {elements.map((e: Object) =>

```

Comment:
THere we can have exact object of TS
If there are not exact type for that, than we can generate custom interface for that boject